python/ert/__init__.py

```diff
@@ -1,8 +1,14 @@
 import warnings
 
-warnings.simplefilter('always', DeprecationWarning) # see #1437
```

Comment:
Should probably do this globally then? There is another one in: libres/python/res/__init__.py

Reply:
Never mind, didn't see that you'd already done that